CHANGELOG-4.1.md

```diff
@@ -8,7 +8,7 @@ To get the diff for a specific change, go to https://github.com/symfony/symfony/
 To get the diff between two versions, go to https://github.com/symfony/symfony/compare/v4.1.0...v4.1.1
 
 * 4.1.4 (2018-08-28)
-
+ * deprecation #28579 [Translator] set the getFallbackLocales() method as deprecated (boscho87)
```

Comment:
This file is autogenerated so all changes should be reverted. But the changelog in the component itself should be updated instead.

Reply:
thus, a PR to master would not change the existing 4.1.4 release